TODO.md

```diff
@@ -0,0 +1,32 @@
+# CNCF Hugo Starter Checklist
+
+## CNCF Website Guidelines
+
+As per the [CNCF Website Guidelines](https://github.com/cncf/foundation/blob/master/website-guidelines.md):
```

Comment:
As much as possible I'm not copying verbatim the guidelines, instead trying to convert what they've put forth in a bit more direct list of things to be done.

Where I am copying verbatim, it's because there is a requirement that some phrase exist on the page, or they've provided a particularly clear example.